google/resource_compute_disk_test.go

```diff
@@ -333,11 +333,47 @@ func TestAccComputeDisk_encryption(t *testing.T) {
 				Config: testAccComputeDisk_encryption(diskName),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckComputeDiskExists(
+						"google_compute_disk.foobar", getTestProjectFromEnv(), &disk),
+					testAccCheckEncryptionKey(
 						"google_compute_disk.foobar", &disk),
+				),
+			},
+		},
+	})
+}
+
+func TestAccComputeDisk_encryptionKMS(t *testing.T) {
```

Comment:
Can you add a note that this test is expected to fail, and going to go away in this provider soon?